README.md

```diff
@@ -72,6 +74,12 @@ SET_COMPLETED_TO_REWATCHING_ON_FIRST_EPISODE=no
 UPDATE_PROGRESS_WHEN_REWATCHING=yes
 SET_TO_COMPLETED_AFTER_LAST_EPISODE_CURRENT=yes
 SET_TO_COMPLETED_AFTER_LAST_EPISODE_REWATCHING=yes
+ADD_ENTRY_IF_MISSING=no
+ANI_CLI_COMPATIBILITY=no
+# Keybind configuration (use MPV keybind format)
+KEYBIND_UPDATE_ANILIST=ctrl+a
+KEYBIND_LAUNCH_ANILIST=ctrl+b
+KEYBIND_OPEN_FOLDER=ctrl+d
```

Comment:
Instead of being an option it should check for `https://` or `http://` in the directory and automatically enable it instead of it being another option.

Also keybinds should probably be left for the user to modify in input.conf